functions/venue.js

```diff
@@ -99,3 +101,24 @@ exports.createVenue = functions.https.onCall(async (data, context) => {
 
   return venueData;
 });
+
+exports.updateVenue = functions.https.onCall(async (data, context) => {
+  if (!context.auth || !context.auth.token) {
+    throw new functions.https.HttpsError("unauthenticated", "Please log in");
+  }
+
+  if (context.auth.token.aud !== PROJECT_ID) {
+    throw new functions.https.HttpsError("permission-denied", "Token invalid");
+  }
+
```

Comment:
this is sth we caopy/paste in all our functions. Might be a good thing to move the function `checkAuthentication` from `payment.js` in a seperate file and use it in all our firebase functions